x-pack/plugins/index_management/__jest__/client_integration/home/home.test.ts

```diff
@@ -83,6 +90,14 @@ describe('<IndexManagementHome />', () => {
         expect(exists('indicesList')).toBe(false);
         expect(exists('templateList')).toBe(true);
       });
+
+      describe('toggles', () => {
+        test('Should show system indices if hidden indices is toggled. ', () => {
+          const { find, actions } = testBed;
+
+          actions.showHiddenIndices();
```

Comment:
There aren't any assertions in this test. We should have two assertions: one before the action to establish the state of the table (which I assume will be that hidden indices _aren't_ listed in the table), and another after the action to establish that the hidden indices _are_ listed in the table.